packages/graphics/graphics.3.09.1/opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 maintainer: "David Allsopp <david.allsopp@metastack.com>"
-bug-reports: "https://github.com/ocaml/ocaml/issues"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
```

Comment:
@dra27 should all these graphics ones redirect to https://github.com/ocaml/graphics/issues instead?

Reply:
For the 5.0.0+ ones, yes. The others are special packages which compile the graphics package from the main distribution and add it, so issues are packaging issues and should be reported to opam-repository. (slightly pedantically, given that I'm a maintainer of both repos........)

Reply:
Thanks! I think this is good to merge then.